deno/src/utils/BroadcastMethods.ts

```diff
@@ -40,7 +40,9 @@ export class BroadcastMethods<T extends Record<string, Function>> {
         const result = await methods[fn](...args)
         chan.postMessage({ result, requestId })
       } catch (error) {
-        chan.postMessage({ error, requestId })
+        try{
+            chan.postMessage({ error, requestId })
+        }catch(e){}
```

Comment:
Is there anyway to match the error for this specific situation as if it's only a development issue the server shouldn't silently fail in production. Also definitely include a `console.error('BroadcastChannel is dead failed to send errror', error)`